engine/src/canvas.mlc

```diff
@@ -3729,6 +3729,10 @@ Description:
 
 >*Note:* The layout bounds is used to position blocks of text relative to each other, so the text may overlap these bounds.
 
+The `layout` is optional and has no effect on the return value, but may be used to emphasize the difference between `the layout bounds` and `the image bounds`.
```

Comment:
I think my intention had always been that LCB syntax should be explicitly one piece of syntax per action - so this does break that for backwards-compatibility reasons.

The 'layout' token was added after we overloaded 'the bounds' to add 'the image bounds'. Perhaps 'the layout bounds' should actually be the syntax, and 'the bounds' should be deprecated?

We can do that easily enough by adding a deprecated syntax definition for 'the bounds' and removing the optionality from 'layout'.